Azure Firewall/Template - Logic app for IDPS signature updates notification/Template.json

```diff
@@ -3,41 +3,74 @@
     "contentVersion": "1.0.0.0",
     "parameters": {
         "Logic_App_Name": {
-            "defaultValue": "IDPSRules-Test1",
+            "defaultValue": "IDPSRulesNotification",
             "type": "String"
         },
-        "connections_office365_1_externalid": {
-            "defaultValue": "/subscriptions/****/resourceGroups/****/providers/Microsoft.Web/connections/office365-1",
-            "type": "String"
+"Sender_Address": {
+            "defaultValue": "",
+            "type": "String",
+  "metadata": {
+                "displayName": "Enter the sender's email address to form the connection to O365 Outlook",
+                "description": "Enter the sender's email address to form the connection to O365 Outlook"
+                }
         },
-		"Email": {
+		"Recipient_Address": {
             "defaultValue": "",
-            "type": "String"
+            "type": "String",
+ "metadata": {
+                "displayName": "Enter the email address to send the IDPS Rules update notification to",
```

Comment:
"displayName": "Enter recipient email address to receive new signature updates",
                "description": "Enter the email address to send the IDPS Signature update notification"

Reply:
Looks good